docs/user-guide/deploying/aws-marketplace.md

```diff
@@ -4,172 +4,109 @@ unlisted: true
+- On the AWS `CloudFormation` page, click `Next` to continue.
 
-#### `CIDRBlock`
-The default (web) ports `80` and `443` are blocked by default for security reasons.
-In this field, you can fill in an `IPv4` address on which you want to enable these ports.
-
-Please make sure you are using CIDR notation:
-`x.x.x.x/32` allow one specific IP address access, or `0.0.0.0/0` allows all IP address access.
-
-## OpenRemote Configuration
-
-#### `Password`
-Create a password for your OpenRemote instance. This password is used for the administrator account.
-The password must meet the following conditions:
-- The minimum length is eight characters
-- Must have at least one special character
-
-## E-mail Configuration
-
-The email configuration is not mandatory. All fields can be left blank / default if email is not required. 
-
-#### `SMTPHost`
-Provide the `SMTP` hostname that you want to use for sending e-mails.
-
-#### `SMTPUser`
-Provide the `SMTP` username that you want to use for sending e-mails.
-
-#### `SMTPPassword`
-Provide the `SMTP` password that corresponds to the hostname and user.
-
-#### `SMTPPort`
-The system is using port `587 (TLS)` by default for sending e-mails. If you want to use something else, for example, `465 (SSL)`, You can change it here.
-
-#### `SMTPTLS`
-Select if you want to use TLS for sending e-mails. Choose between `true` or `false`.
+## Instance Configuration
+If you are successfully subscribed to the OpenRemote marketplace `app`, you can start configuring it. Below, you will find a detailed description of each parameter available in the template.
 
-#### `SMTPFrom`
-Provide the e-mail address that you want to use for sending e-mails. The e-mail address must be usable by the `SMTP` host.
+- **Would you like to use your own domain name? If so, please enter it here.** <br/>
+You can specify the `FQDN (Fully Qualified Domain Name)` you want to use for this OpenRemote instance.  
+If no value is provided, you can access OpenRemote using the public `IPv4` address of the `EC2` instance.
 
-#### `SMTPProtocol`
-Select the SMTP protocol you want to use for sending e-mails. Choose between `smtp` or `smtps` (smtps = SSL).
+- **Which instance would you like to use?** <br/>
+You can choose from the following `t4g` and `m6g` instance types.
+  - `t4g.small`
+     - vCPU: 2 
+     - Memory: 2GB
+  - `t4g.medium` 
+     - vCPU: 2 
+     - Memory: 4GB
+  - `t4g.large` 
+     - vCPU: 2 
+     - Memory: 8GB
+  - `m6g.large` 
+     - vCPU: 2 
+     - Memory: 8GB
+  - `m6g.xlarge` 
+     - vCPU: 4 
+     - Memory: 16GB
+  
+   Prices vary based on the selected instance. All instances are using the `ARM` architecture. <br/>
+   For detailed pricing information, visit the pricing pages for [t4g](https://aws.amazon.com/ec2/instance-types/t4/) and [m6g](https://aws.amazon.com/ec2/instance-types/m6g/).
 
-# Updating 
+- **How much storage would you like to allocate to the virtual machine?** <br/>
```

Comment:
When writing Markdown HTML tags should only be used if absolutely necessary. Usually `<br/>` can be replaced by using new lines.

```suggestion
- **How much storage would you like to allocate to the virtual machine?**

```